src/frontend/src/components/Table/index.tsx

```diff
@@ -71,7 +71,10 @@ const Table = <T extends any>({
     data,
     defaultColumn: {
       minSize: 50,
-      size: 50,
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+      // @ts-ignore - (mlila) this is a fix for a table spacing problem we had
+      // https://github.com/TanStack/table/discussions/3192
+      size: "auto",
```

Comment:
Will this @ts-ignore still work since it's 2 lines up? Just wondering if prettier wrapped this comment

Reply:
it does work! Otherwise the linter would fail 😅